res/database/Default/database_programmes_fictional.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
should this file be user/fronck.xml?

Reply:
so far there is no fronck.xml file so it would be the only entry, hence adding it into this file where the other fictional titles will be moved in the next PR.

Reply:
OK.